Header header
time gpst
float32 ec
int32 ppm
float32 salinity
float32 specificGrav
